docs/geedocs/answer/7160000.html

```diff
@@ -148,10 +149,95 @@
           <td>Close the resource form and open it again to make the save option available again.</td>
         </tr>
         <tr>
+          <td>340</td>
+          <td>GE Fusion Terrain is black</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
           <td>342</td>
           <td>Fusion crashes when opening an unsupported file type</td>
           <td>Re-open fusion and avoid opening unsupported file types.</td>
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
+        </tr>
+        <tr>
+          <td>380</td>
+          <td>Provider field in resource-view is blank</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>401</td>
+          <td>GEE commands are not in the path for sudo. </td>
+          <td>Add /opt/google/bin to the path for all users, including the super user</td>
+        </tr>
+        <tr>
+          <td>402</td>
+          <td>Provider manager window locked to main window.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>403</td>
+          <td>Missing close button on system manager window in RHEL 7</td>
+          <td>Right click the title bar and select close</td>
+        </tr>
+        <tr>
+          <td>404</td>
+          <td>Opaque polygons in preview.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>405</td>
+          <td>Vector layer preview not cleared in some situations</td>
+          <td>Reset the preview window to the correct state by either clicking on it or previewing another vector layer</td>
+        </tr>
+        <tr>
+          <td>407</td>
+          <td>Corrupt data warning when starting fusion</td>
+          <td>No current work around</td>
```

Comment:
May want to state here that Fusion still operates correctly (as far as we can tell) so you can ignore the warning.

Reply:
ok. How about 'No current work around but Fusion loads and runs correctly' ?

Reply:
Sounds good.